### PR TITLE
Add the code blocks in authorization.md

### PR DESCRIPTION
**Pull request by ydFu**
* Add the code blocks in the Markdown spec to make it easy to read.

* Add description that distinguish between **command** and **output** make it easy to read.

* Adjust description in Kubernetes components for smoother reading.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Comment by tengqm** [comment]
@ydFu Please try propose some meaningful improvements instead. I'm not saying that this one is useless. There are values in it, but the improvement is so trivial and it is combined with invalid modifications... Think twice?

**Comment by ydFu** [comment]
In my learning journey for per doc,by the way I improve the doc to contribute to the community.
I will adjust how I contribute.Thanks for the reminder. @tengqm

File: content/en/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Kubernetes sometimes checks authorization for additional permissions using speci
 
 The Kubernetes API server may authorize a request using one of several authorization modes:
 
- * **Node** - A special-purpose authorization mode that grants permissions to kubelets based on the pods they are scheduled to run. To learn more about using the Node authorization mode, see [Node Authorization](/docs/reference/access-authn-authz/node/).
+ * **Node** - A special-purpose authorization mode that grants permissions to the kubelet based on the pods they are scheduled to run. To learn more about using the Node authorization mode, see [Node Authorization](/docs/reference/access-authn-authz/node/).
```
Comment (tengqm) [inline content/en/docs/reference/access-authn-authz/authorization.md:90]: I don't think the above change is right

File: content/en/docs/reference/access-authn-authz/authorization.md [hunk 2]
```diff
@@ -176,7 +185,7 @@ The following flags can be used:
   * `--authorization-mode=ABAC` Attribute-Based Access Control (ABAC) mode allows you to configure policies using local files.
   * `--authorization-mode=RBAC` Role-based access control (RBAC) mode allows you to create and store policies using the Kubernetes API.
   * `--authorization-mode=Webhook` WebHook is an HTTP callback mode that allows you to manage authorization using a remote REST endpoint.
-  * `--authorization-mode=Node` Node authorization is a special-purpose authorization mode that specifically authorizes API requests made by kubelets.
+  * `--authorization-mode=Node` Node authorization is a special-purpose authorization mode that specifically authorizes API requests made by the kubelet.
```
Comment (tengqm) [inline content/en/docs/reference/access-authn-authz/authorization.md:188]: this one also

**Comment by bradtopol** [comment]
Assigning @tengqm since he is alread engaged on this.
/assign @tengqm

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25982--kubernetes-io-master-staging.netlify.app/docs/reference/access-authn-authz/authorization/

**Comment by kbhawkey** [comment]
Thanks @ydFu .
/approve

**Pull request merged**